src/cmd/split.h

```diff
@@ -91,69 +92,76 @@ char* strdup_with_new(const char* the_string) {
 // See //util/csv/parser.h for more complete documentation.
 //
 // ----------------------------------------------------------------------
-void SplitCSVLineWithDelimiter(char* line, char delimiter, std::vector<char*>* cols) {
```

Comment:
Add unit test for SplitLineWithDelimiter.
- empty line
- delimiter `' '`, `','`, `'---'`, `''`, etc